### DISCUSSION TITLE
Pick specific measuring system/units for each metric in live timing

### DISCUSSION THREAD

**Discussion by Walker1390** [f1multiviewer/issue-tracker#81] [Feature requests] [answered]
This FR might only help those of us living in the UK who still use/are used to the 'pick and mix' of metric and imperial measurements. For example for speeds and distances while driving we still use mph and miles but use pretty much exclusively use celcius for temperature. Don't get me wrong the metric system is far superior but its not as easy to get a feel for the speed of a car at x km/h if you've never used that system for vehicle speeds before. Just a quality of life suggestion.

**Comment by f1multiviewer** [comment]
Would it be OK to just add another "option", which would work for the majority of the UK? Adding separate options may be a bit much

**Comment by JJWatMyself** [comment]
I agree that this will be good for us Brits.  Third option: British (mph, meters, celsius).

**Comment by f1multiviewer** [comment, marked as answer]
<img width="292" alt="image" src="https://user-images.githubusercontent.com/106821994/176976111-7f5541ac-521d-46e0-b813-0893594ce6f3.png">

Ships in the next release!

**Reply by JJWatMyself** [reply]
Rock star!
